jsonschema-maven-plugin/README.md

```diff
@@ -0,0 +1,133 @@
+# Java JSON Schema Generator – Maven Plugin
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.github.victools/jsonschema-module-javax-validation/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.github.victools/jsonschema-maven-plugin)
+
+Maven plugin for the [jsonschema-generator](../jsonschema-generator) – Integrating JSON Schema generation in your builds
+
+## Features
+1. Generate the JSON schema for a Java class.
+2. Configure the generator by selecting the modules to be used.
+3. Configure the options of the modules used in the generation.
+4. Configure the use of custom modules.
+
+## Usage
+### Plugin definition
+```xml
+<plugin>
+    <groupId>com.github.victools</groupId>
+    <artifactId>jsonschema-maven-plugin</artifactId>
+    <version>4.8.0</version>
+    <executions>
+        <execution>
+            <goals>
+                <goal>schema-generator</goal>
+            </goals>
+        </execution>
+    </executions>
+    <configuration>
+        <className>com.myOrg.myApp.myclass</className>
+    </configuration>
+</plugin>
+```
+This will use the default configuration of the generator. This will enable all of the modules that are part of the generator. 
+
+
+### Configuring file locations
+```xml
+<configuration>
+    <className>com.myOrg.myApp.myclass</className>
+    <schemaFileName>mySchema.json</schemaFileName>
+    <schemaFilePath>src/main/resources/schemas</schemaFilePath>
+</configuration>
+```
+- The default name of the file is the name of the class extended with ".schema.json"
+- The default path is "src/main/resources"
+### Configuring schema version
+The version of JSON Schema that is to be used can be configed with the schemaVersion element.
+```xml
+<configuration>
+    <className>com.github.victools.jsonschema.plugin.maven.TestClass</className>
+    <schemaVersion>DRAFT_2019_09</schemaVersion>
+</configuration>
+```
+Allowed values are:
+- DRAFT_2019_09
+- DRAFT_7
+
+The last one is the default
```

Comment:
I recently added `DRAFT_6` as well, since there was hardly any relevant difference to `DRAFT_7`.
```suggestion
Allowed values are:
- `DRAFT_2019_09`
- `DRAFT_7`
- `DRAFT_6`

`DRAFT_7` is the default, if not specified.

```

Reply:
ok